setup.py

```diff
@@ -63,6 +64,7 @@
         "console_scripts": [
             "csv2ecl=ecl2df.csv2ecl:main",
             "ecl2csv=ecl2df.ecl2csv:main",
+            "ecl2arrow=ecl2df.ecl2csv:main",
```

Comment:
`ecl2arrow` == `ecl2csv` in the terminal? Does that mean that `ecl2arrow` by default gives `csv`?

Reply:
There is a trick here that solves that: https://github.com/equinor/ecl2df/pull/358/files#diff-6eaf200e2166a36db9538a8a79197b6a69d080a1046ea080600c8601f5182e2cR289